scripts/setup-updates

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/python3
```

Comment:
Thanks for doing this. It will help to clear the main script which is still too messy and hard to find anything in it.
Even if we don't need this in a future it is nice to have this as a separate script.

But wouldn't this be in conflict with the old variant when this is added but the old is not removed?

Reply:
The code seems to check for the folder to already exist before doing anything, so it should be fine.
My theory is that some other tool (Lorax ?) might be (or might have been) doing the updates/product image setup and the code needs to be able to handle that.

Reply:
Sounds reasonable.